packages/core/test/unit/initializer/stryker-initializer.spec.ts

```diff
@@ -55,7 +56,8 @@ describe(StrykerInitializer.name, () => {
     syncBuiltinESMExports();
     sut = testInjector.injector
       .provideValue(initializerTokens.out, out as unknown as typeof console.log)
-      .provideValue(initializerTokens.restClientNpm, npmRestClient)
+      .provideFactory(initializerTokens.npmRegistry, getRegistry)
+      .provideFactory(initializerTokens.restClientNpm, createNpmRegistryClient)
```

Comment:
Since this is a unit test for the `StrykerInitializer`, I wouldn't expect the "real" `getRegistry` to be used here. Instead, you can keep using `provideValue` here for the `npmRestClient` sinon mock here. 

Indeed, the tests are failing, see https://github.com/stryker-mutator/stryker-js/actions/runs/10341544430/job/28626286827?pr=4937